src/administration/Administration.Service/Controllers/CompanyDataController.cs

```diff
@@ -275,11 +291,16 @@ public async Task<NoContentResult> CreateSsiCertificate([FromForm] SsiCertificat
     /// <param name="cts">Cancellation Token</param>
     /// <returns>No Content</returns>
     /// <response code="204">Successfully approved the credentials.</response>
+    /// <response code="404">CompanySsiDetail does not exists</response>
+    /// <response code="409">Credential is in Incorrect State</response>
```

Comment:
Please use only one entry for each response code. This applies to the other changes as well

you could format it like:

```suggestion
    /// <response code="409">
    /// Credential is in Incorrect State <br />
    /// VerifiedCredentialTypeKindId must not be null
    /// </response>
```